src/Youtube-Scraper.js

```diff
@@ -67,11 +67,10 @@ class CommentScraper {
       }
     }
 
-    let total = null;
+    let total = null
     if (!continuation) {
-        const headerElem = commentPageResponse.data.onResponseReceivedEndpoints[0].reloadContinuationItemsCommand.continuationItems[0]
-        if ('commentsHeaderRenderer' in headerElem)
-        total = Number(headerElem?.commentsHeaderRenderer?.countText?.runs?.[0]?.text?.replace(',', '')) ?? null
+      const headerElem = commentPageResponse.data.onResponseReceivedEndpoints[0].reloadContinuationItemsCommand.continuationItems[0]
+      if ('commentsHeaderRenderer' in headerElem) { total = Number(headerElem?.commentsHeaderRenderer?.countText?.runs?.[0]?.text?.replace(',', '')) ?? null }
```

Comment:
Why isn't this considered as a long line?
```suggestion
      if ('commentsHeaderRenderer' in headerElem) { 
        total = Number(headerElem?.commentsHeaderRenderer?.countText?.runs?.[0]?.text?.replace(',', '')) ?? null
      }
```